scripts/beekeeper.pl

```diff
@@ -507,6 +514,49 @@ sub register_beekeeper {
     return $beekeeper;
 }
 
+
+sub big_red_button {
+  my ( $self, $valley ) = @_;
+
+  my $bk_a = $self->{dba}->get_Beekeeper();
```

Comment:
The convention is to call `get_BeekeeperAdaptor` (the only exception is Queen, for historical reasons)

Reply:
Ah, historic reasons. Fixed.